docs/SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, please contact the Security Team: `cncf-kubeedge-security@lists.cncf.io`. Security is of the highest importance and we sincerely request you to keep the vulnerability information confidential.
+
+The team will help diagnose the severity of the issue and determine how to address the issue. The reporter(s) can expect a response within 24 hours acknowledging the issue was received. If a response is not received within 24 hours, please reach out to any committer directly to confirm receipt of the issue.
+
+## Review Process
+
+After receiving any suspected vulnerability, the Security Alert Team will discuss the issue with the reporter(s) and KubeEdge's security advisors to analyze/validate the vulnerability, assess its severity based on its actual impact on KubeEdge.
+
+If the vulnerability is accepted, its remediation priority, and develop remediations (including mitigations, patches/versions, and other risk mitigations) will be determined.
+
+Sticking to the principles of harm and risk reduction, we release vulnerability information to stakeholders through the security announce mailing list (`cncf-kubeedge-distributors-announce@lists.cncf.io`). The reporter(s) are expected to participate in the discussion of the timeline and abide by agreed upon dates for public disclosure. 
+
+Note also that the team can use [GitHub Security Advisories](https://github.com/kubeedge/kubeedge/security/advisories) to disclose, fix, and publish information about the vulnerability.
```

Comment:
It's needed to splite the `Distributors Disclosure` and `Public Disclosure` here. First disclosure to distributors then the public disclosure